src/transport/transport.module.ts

```diff
@@ -3,52 +3,68 @@ import { Kafka, logLevel } from 'kafkajs';
 import { WINSTON_MODULE_NEST_PROVIDER } from 'nest-winston';
 import { Configuration } from 'common/config';
 import { TransportInterface } from './transport.interface';
-import { KafkaTransport } from './kafka.transport';
-import { KAFKA_LOG_PREFIX } from './kafka.constants';
+import { KafkaTransport } from './kafka/kafka.transport';
+import { KAFKA_LOG_PREFIX } from './kafka/kafka.constants';
 import { WalletModule, WalletService } from '../wallet';
+import RabbitTransport from './rabbit/rabbit.transport';
+import RabbitClient from './rabbit/rabbit.client';
+import { FetchModule, FetchService } from '@lido-nestjs/fetch';
 
 export type SASLMechanism = 'plain' | 'scram-sha-256' | 'scram-sha-512';
 
 @Module({
   exports: [TransportInterface],
-  imports: [WalletModule],
+  imports: [WalletModule, FetchModule.forFeature()],
```

Comment:
It's better to think about enabling middleware for `FetchModule` and including Prometheus metrics here.

Reply:
Can you provide some example? (link to code)